airflow/cli/commands/db_command.py

```diff
@@ -50,6 +50,16 @@ def upgradedb(args):
     db.upgradedb()
 
 
+@cli_utils.action_logging
```

Comment:
This decorator writes a row to the (audit) log table -- and there is a good chance that this table won't exist when this command is run, so we shouldn't have this decorator here.

```suggestion
```